test/fixtures/controller.line/cubicInterpolationMode/scriptable.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+	config: {
+		type: 'line',
+		data: {
+			labels: [0, 1, 2, 3, 4, 5],
+			datasets: [
+				{
+					// option in dataset
+					data: [0, 4, 2, 6, 4, 8],
+					borderColor: '#ff0000',
+					cubicInterpolationMode: function(ctx) {
+						var mode = ctx.datasetIndex === 0 ? 'monotone' : 'default';
+
+						return mode;
+					}
+				},
+				{
+					// option in element (fallback)
+					data: [0, 4, 2, 6, 4, 8],
```

Comment:
We shouldn't use the same data because both lines are on top of each other, making the test way less robust. We should make sure that lines are not overlapping. Same for all `cubicInterpolationMode` tests.